virtManager/delete.py

```diff
@@ -524,6 +532,9 @@ def _get_title_text(self, devs):
         text = _('%s') % self.disk.target
         return [title, text]
 
+    def _vm_active_status(self)
```

Comment:
This should always return False. We do not want to show a warning about the active VM when just removing storage.